code/modules/reagents/reagent_dispenser.dm

```diff
@@ -75,22 +77,25 @@
 	else
 		qdel(src)
 
+/obj/structure/reagent_dispensers/proc/tank_leak()
+	if(leaking && reagents && reagents.total_volume >= amount_to_leak)
+		reagents.expose(get_turf(src), TOUCH, amount_to_leak / max(amount_to_leak, reagents.total_volume))
+		reagents.remove_reagent(reagent_id, amount_to_leak)
+		return TRUE
```

Comment:
```suggestion
		return TRUE
	return FALSE
```

```suggestion
```

Generally, everything should return a value or shouldn't return a value. Pick one.